pages/common/docker-stats.md

```diff
@@ -0,0 +1,24 @@
+# docker stats
+
+> Display a live stream of containers resource usage statistics.
+> More information: <https://docs.docker.com/engine/reference/commandline/stats/>.
+
+- Display live stream of all running containers statistics:
+
+`docker stats`
+
+- Display live stream statistics on multiple containers by name or id:
+
+`docker stats {{container_name container_id ...}}`
```

Comment:
Based in the `docker container` we can use:
```suggestion
`docker stats {{container_name}}`
```
This command is advanced, I think the user knows they can use the name or the ID.